js/kaltura-playlist-control.js

```diff
@@ -105,8 +109,67 @@
         var _hideLoader = function () {
             jQuery('.playlist-loader').hide();
         };
+        var _playlistInit = function () {
+            playlistBox.nanoScroller({ contentClass: 'kaltura-nano-playlist-content' });
+
+        };
+        var _playlistBindScroll = function () {
+            playlistBox.bind("scrollend", function (e) {
+                var page = playlistBox.data('page');
+                var pageSize = playlistBox.data('page-size');
+                var loading = playlistBox.data('loading');
+                if (loading == false) {
+                    playlistBox.data('loading', true);
```

Comment:
`playlistBox.data('loading', true);` can be removed as it is also added in `_fetchPlaylists`